docs/operator-manual/custom-styles.md

```diff
@@ -118,3 +118,26 @@ data:
 ```
 
 ![banner with link](../assets/banner.png)
+
+## Disabling "All" Option in Items Per Page
+
+For large ArgoCD instances with thousands of applications, allowing users to select "all" items per page can cause significant UI performance issues. You can disable the "all" option from the items per page dropdown by setting `ui.paginateAllEnabled` to `false` in the `argocd-cm` ConfigMap.
```

Comment:
```suggestion
For large ArgoCD instances with thousands of applications, allowing users to select "all" items per page can cause significant UI performance issues. You can disable the "all" option from the items per page dropdown by setting `ui.paginateAllDisabled` to `true` in the `argocd-cm` ConfigMap.
```

The snippet already uses `ui.paginateAllDisabled`, so this wording keeps the description consistent?